pkgs/development/libraries/libmd/default.nix

```
@@ -1,14 +1,26 @@
-{ lib, stdenv, fetchurl, autoreconfHook }:
+{ lib, stdenv, fetchurl, fetchpatch, autoreconfHook }:
 
 stdenv.mkDerivation rec {
   pname = "libmd";
   version = "1.0.4";
 
   src = fetchurl {
-    url = "https://archive.hadrons.org/software/${pname}/${pname}-${version}.tar.xz";
+    urls = [
+      "https://archive.hadrons.org/software/libmd/libmd-${version}.tar.xz"
+      "https://libbsd.freedesktop.org/releases/libmd-${version}.tar.xz"
+    ];
     sha256 = "sha256-9RySEELjS+3e3tS3VVdlZVnPWx8kSAM7TB7sEcB+Uw8=";
   };
 
+  patches = lib.optionals stdenv.isDarwin [
+    # Drop aliases for SHA384 functions, because such aliases are not supported on Darwin.
+    (fetchpatch {
+      url = "https://raw.githubusercontent.com/macports/macports-ports/8332f5dbcaf05a02bc31fbd4ccf735e7d5c9a5b0/devel/libmd/files/patch-symbol-alias.diff";
```

GitHub URL makes it easier to visit the repo:

```suggestion
      url = "https://github.com/macports/macports-ports/raw/8332f5dbcaf05a02bc31fbd4ccf735e7d5c9a5b0/devel/libmd/files/patch-symbol-alias.diff";
```